components/vc/pdu/include/HW/HW_gpio_componentSpecific.h

```diff
@@ -9,12 +9,87 @@
  *                             T Y P E D E F S
  ******************************************************************************/
 
-typedef enum
+
+typedef enum 
 {
-    HW_GPIO_CAN1_RX = 0U,
-    HW_GPIO_CAN1_TX,
+    HW_GPIO_BMS2_EN = 0U,
+    HW_GPIO_ACCUM_EN,
+    HW_GPIO_LP2_LATCH,
+    HW_GPIO_BSPD_MEM,
+    HW_GPIO_CS_SD,

```

Comment:
Rename to `HW_GPIO_SPI_NCS_SD`. It is the chip select of the sd card on the spi bus